CHANGELOG.md

```diff
@@ -10,3 +10,6 @@
 - BASE : Cache memory performace improved by not stocking geographical entities anymore
 
 ## Bug Fixes
+
+## Changes
+- Removed block persmissions from demo data
```

Comment:
"Changes" section in wip-6.0 corresponds to "Improvements" in other branches.
Also please fix typo "persmissions" => "permission"